tools/firenear/motd_node_manager

```diff
@@ -8,12 +8,12 @@
               /_/   \_,_//___/\__/
 
 Welcome to this node manager pod, common commands:
- - Show version                       sfnear --version
+ - Show version                       firenear --version
 
 Node manager commands (near-... <tab> to show all):
  - Check if running                   near-is-running
 
- - Debug Deep Mind logs for 30s       near-debug-deep-mind-30s
+ - Debug Firehose logs for 30s       near-debug-firehose-30s
```

Comment:
`near-debug-firehose-30s` must fit with your rename of `tools/firenear/scripts/near-debug-deep-mind-30s`